src/core/NotePlayHandle.cpp

```diff
@@ -161,7 +175,7 @@ void NotePlayHandle::setVolume( volume_t _volume )
 
 	const int baseVelocity = m_instrumentTrack->midiPort()->baseVelocity();
 
-	m_instrumentTrack->processOutEvent( MidiEvent( MidiKeyPressure, midiChannel(), midiKey(), midiVelocity( baseVelocity ) ) );
+	m_instrumentTrack->processOutEvent( MidiEvent( MidiKeyPressure, midiChannel(), key(), midiVelocity( baseVelocity ) ) );
```

Comment:
```suggestion
	m_instrumentTrack->processOutEvent(MidiEvent(MidiKeyPressure, midiChannel(), key(), midiVelocity(baseVelocity)));
```
Should `key` be renamed to `midiKey`?

Reply:
Oh wait, I just realized, `key()` is a inherited method from `Note`, so you can't exactly rename it. But I could add an extra `midiKey()` method which simply returns `key()`.